helix-term/src/compositor.rs

```diff
@@ -79,6 +79,7 @@ pub struct Compositor {
     area: Rect,
 
     pub(crate) last_picker: Option<Box<dyn Component>>,
+    pub(crate) full_redraw: bool,
```

Comment:
Shall we put this in `Compositor` or `Editor`, alongside the `redraw_handle` and `need_redraw` flags?